content/technologies/ai.mdx

```diff
@@ -7,4 +7,4 @@ readMoreSlug: 'https://www.ssw.com.au/consulting/artificial-intelligence'
 
 ### Bots/AI
```

Comment:
5. Bots/AI - any internal links we should add here? https://www.ssw.com.au/consulting/artificial-intelligence

Reply:
done